lib/package/commands.coffee

```diff
@@ -41,9 +41,9 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.eval(cell: true, move: true)
-      'julia-client:next-cell': =>
+      'julia-client:next-cell': ->
```

Comment:
I'd prefer keeping these as fat arrows (just for consistency with the JS code).

Reply:
Sure I revert the commit soon 😉